OneAndOnlyOdin/Re-Locked
# Add Italian locales

- Add Italian locales

How to test:
- Visit chrome://settings/languages in Chrome
- Add Italian as a language and move it to the top
- Open the frontend in Chrome it should be displayed in Italian
